src/types/redis_hyperloglog.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+#pragma once
+
+#include "storage/redis_db.h"
+#include "storage/redis_metadata.h"
+
+namespace redis {
+
+class HyperLogLog : public Database {
+ public:
+  explicit HyperLogLog(engine::Storage *storage, const std::string &ns) : Database(storage, ns) {}
+  rocksdb::Status Add(const Slice &user_key, const std::vector<uint64_t> &element_hashes, uint64_t *ret);
+  rocksdb::Status Count(const Slice &user_key, uint64_t *ret);
+  rocksdb::Status Merge(const std::vector<Slice> &user_keys);
```

Comment:
hmm I didn't find it's definition 🤔

Reply:
Aha, I've update the pr description, I remove merge related code to make it simple, I'll add it back in next pr